.pre-commit-config.yaml

```diff
@@ -49,5 +52,5 @@ repos:
           - --max-line-length=120
           - --min-public-methods=0
           - --good-names=q,f,fp,i,e
-          - --disable=E0401,W1201,W1203,C0114,C0115,C0116,C0411,W0107,W0702,R0801,R1705,R1710
+          - --disable=E0401,W1201,W1203,C0114,C0115,C0116,C0411,W0107,W0702,R0801,R1705,R1710,W0718,W0719
```

Comment:
I would rather try avoiding adding these 2, as it might lead to temptation of not paying attention everywhere from this point onward.
Would it be possible to fix the code (especially the `raise` one), and when not practical, then disabling the test for that specific code line (i.e.:  `# pylint: disable=W0718`  on the `except` line) ?